app/client/src/api/ActionAPI.tsx

```
@@ -233,17 +231,10 @@ class ActionAPI extends API {
-    parentSpan?: OtlpSpan,
   ): Promise<AxiosPromise<ActionExecutionResponse>> {
     ActionAPI.abortActionExecutionTokenSource = axios.CancelToken.source();
 
-    if (!parentSpan) {
-      return this.executeApiCall(executeAction, timeout);
-    }
-
-    return wrapFnWithParentTraceContext(parentSpan, async () => {
-      return await this.executeApiCall(executeAction, timeout);
-    });
+    return await this.executeApiCall(executeAction, timeout);
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)